script/Ciel/Day 9/09_01_CIEL09_2B.txt

```diff
@@ -113,150 +122,170 @@ Arcueid Brunestud is a True Ancestor.
 -- Page 166, Offset 22547.
 -- 「……消滅した後も隷属死徒たちが主の肉体を修復し、力ある血液を注ぎこめば復元はするでしょうが、完全に消滅した後、ひとりでに蘇生する吸血鬼はいない。
-"Even when they have perished, they can still recover if their vassal Dead Apostles mend the body and infuse it with potent blood. There is no vampire that can resurrect from total annihilation on their lonesome.
+"Even after they've perished,
+ it's possible for vampires to recover if their vassal Dead Apostles repair their body and infuse it with potent blood,
+ but there are no vampires that can resurrect from total annihilation on their own.
 }
 [sha:eba08e27a1a0117c5a31fa274473093d64a471a2]{
 -- Page 166, Offset 22548.
 -- けれどロアはそれすら克服してしまった吸血鬼なんです」
-However, Roa is a vampire who has even overcome that hurdle."
+That said, Roa is a vampire who has managed to overcome even that hurdle."
 }
 [sha:e06702c4f11d5325e1e77bc8c368b9823747fe74]{
 -- Page 167, Offset 22549. Mods: @n.
 -- 「遠野くんは輪廻転生という言葉をご存知ですか？
-"Tohno-kun, have you heard about the circle of transmigration?
+// 輪廻転生 prev Samsara in Ciel 07_01_CIEL07_5
+"Tohno-kun, are you familiar with the word Samsara?
 }
 [sha:0995b4e0599c4785f6c2c5da84d5f01e90da0b1e]{
 -- Page 167, Offset 22550. Glued.
 -- 仏教用語ですから日本の方には馴染みはあると思うんですけど」
-It is a Buddhist concept, but I believe that the Japanese are familiar with it."
+ It's a Buddhist concept, so I'd expect most people in Japan to have at least heard of it."
 }
 [sha:9c29fb6d60d8a0dbf9bf4040fd27243ac9a39875]{
 -- Page 167, Offset 22551.
 -- ……輪廻転生。
-The circle of transmigration.
+Samsara...
 }
 [sha:2447f255910dced752c8583929734bbf86a7cb9b]{
 -- Page 167, Offset 22552.
 -- アルクェイドも言っていたロアの特性……。
-Arcueid also mentioned this characteristic of Roa...
+// See offset 15921 and friends
+Arcueid mentioned this was one of his defining traits too.
 }
 [sha:213a740d98d6e49239fd2ba66ea60429ef60bce7]{
 -- Page 168, Offset 22553.
 -- 「……はい、ありますけど。死んでも次の人間として生まれ変わるっていうヤツでしょう」
-"Yes, I have heard of it. It states that one will be reincarnated as another human after their death."
+"Yeah, I've heard of it... It's the idea that you'll be reincarnated as someone else after you die, right?"
 }
 [sha:081807118a6cce039151bb6beb5976d770a7dc5b]{
 -- Page 168, Offset 22554.
 -- 「ロアの特性はまさにそれです。
-"This is exactly what characterizes Roa.
+"And that concept is precisely what makes Roa so special.
 }
 [sha:4c8273d87e737cc5661ce4fdab42b3c755d4e048]{
 -- Page 168, Offset 22555.
 -- 彼は輪廻転生を魔術として成功させた。死んでも生き返る、殺しても殺せない、とはそういう意味なんです」
-He has successfully entered the circle of transmigration through the use of magecraft, and reincarnates after death. Unable to die, even if he is slain."
+// Did he _enter into_ an existing (global) cycle of Samsara? Or make his own version of Samsara, with blackjack?
+// Assuming latter, since we don't really see anyone else reincarnating by default.
+He has succeeded in using magecraft to emulate the cycle of Samsara.
+ Even if he dies, he comes back to life.
+ Even if he is killed, he cannot be slain."
```

Comment:
Prefer it as is imo. It's ridiculous and two lines either way, and existing preserves the ~ても (emphasizing that _despite actions taken against him_ he'll come back) instead of just making it a definite action